admin/components/Post/ArticleCreate.js

```diff
@@ -15,58 +45,26 @@ export default class CreateArticle extends Component {
         isMarkdown: false
     };
 
-    componentWillUnmount() {
-        document.body.classList.remove("options-open");
-    }
-
-    changeEditor = e => {
-        const mode = e.target.checked ? "markdown" : "rich-text";
-        PostActions.setData({ mode });
-        this.setState({ isMarkdown: ~~e.target.checked });
-    };
-
     render() {
         return (
-            <div className="card">
-                <article className="post">
+            <React.Fragment>
```

Comment:
You can import Fragment like: `import React, { Component, Fragment } from "react";` and use just `<Fragment>`.